pkg/render/apiserver.go

```diff
@@ -354,6 +365,18 @@ func (c *apiServerComponent) apiServiceAccountClusterRole() *rbacv1.ClusterRole
 		})
 	}
 
+	if c.installation.CertificateManagement != nil {
```

Comment:
So our API server was never registered as an API server with k8s?

Reply:
This comment seems a bit out of place, but I can tell that you responded to - now - outdated code. The tigera-apiserver _was_ and _is_ registered by the operator as an apiService. This is a special object for aggregated apiservers.

That being said there was a brief moment where this PR was not registering it for Cert management, but that change has been rolled back as we discussed earlier this week.